components/post_view/message_attachments/action_menu/index.ts

```diff
@@ -27,11 +31,15 @@ function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
     };
 }
 
-const mapDispatchToProps = {
-    selectAttachmentMenuAction,
-    autocompleteChannels,
-    autocompleteUsers,
-};
+function mapDispatchToProps(dispatch: Dispatch) {
```

Comment:
The above (removed lines / plain object) give exactly the same functionality in a simpler way.

Reply:
Agree; the other way is simpler and preferred.